driver/kubernetes/factory.go

```diff
@@ -117,6 +118,44 @@ func (f *factory) New(ctx context.Context, cfg driver.InitConfig) (driver.Driver
 				}
 			}
 			deploymentOpt.NodeSelector = s
+		case "tolerations":
+			ts := strings.Split(strings.Trim(v, `"`), ";")
+			deploymentOpt.Tolerations = []corev1.Toleration{}
+			for i := range ts {
+				kvs := strings.Split(ts[i], ",")
+				if len(kvs) == 0 {
+					return nil, errors.Errorf("Invalid tolaration %q", v)
```

Comment:
fmt.Errorf

Reply:
`errors` is correct but lowercase the error

Reply:
Error lowercased